### PR TITLE
Docs/vuejsupskilling

### PR DESCRIPTION
**Pull request by AmosSaji**
Learning Javascript and vue framework necessary for the upskilling for the front-end contributions.

Fixes # (issue)

## Type of change

_Please delete options that are not relevant._

- [ ] Documentation (update or new)

# How Has This Been Tested?
Github tests

## Testing Checklist:

- [ ] Tested in latest Chrome
- [ ] Tested in latest Safari
- [ ] Tested in latest Firefox

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from ... on the Pull Request

### CODE REVIEW

File: docs/ArtGallery/FrontEndUpgrades/amos_html.md
Comment (AmosSaji) [inline docs/ArtGallery/FrontEndUpgrades/amos_html.md:1]: this is html and css file. there is a mistake in the description in the pull request. it was mistakenly written as js and vue description. Kindly check this comment before reviewing the file. thanks

**Comment by AmosSaji** [comment]
[Learn VUE script for front-end](https://trello.com/c/XviBmaF7/171-learn-vue-script-for-front-end)

**Comment by AmosSaji** [comment]
[Learn VUE script for front-end](https://trello.com/c/XviBmaF7/171-learn-vue-script-for-front-end)

**Comment by maddernd** [comment]
Closing PR, no review.

**Pull request closed without merging**